code/modules/clothing/under/suits.dm

```diff
@@ -31,14 +31,22 @@
 	item_color = "waiter"
 	can_adjust = FALSE
 
-/obj/item/clothing/under/suit/black
-	name = "black suit"
+/obj/item/clothing/under/suit/blacktwopiece
+	name = "black two piece suit"
 	desc = "A black suit and red tie. Very formal."
 	icon_state = "black_suit"
 	item_state = "bl_suit"
 	item_color = "black_suit"
 	can_adjust = FALSE
 
+/obj/item/clothing/under/suit/black
+	name = "black suit"
+	desc = "A professional looking black suit. Ready for some serious law."
```

Comment:
should there be a hyphen between "professional" and "looking" (I'm genuinely asking here, as I don't know)